docs/source/use_cases/annotate_variants/interval_joins.rst

```diff
@@ -17,91 +17,84 @@ __ RenderedVersion_
 
 .. comment: end: goto-read-the-docs
 
-If you want to use BigQuery to JOIN variants with other data described by genomic region intervals (overlaps), this page offers a technique to reduce the cross-product and speed up the query by applying `Google BigQuery user-defined functions`_.
+If you want to use BigQuery to JOIN variants with other data described by genomic region intervals (overlaps), this page demonstrates the use of a complex JOIN predicate.
 
 Example
 -------
 
 Let's use a concrete example: *Suppose you have a list of genes names and you want to find all the rare SNPs overlapping those genes and also 100,000 bp on either side of the gene for all of your whole genome samples.*
```

Comment:
I don't think the "Suppose  you have a list" example needs to be italicized. I would format it just as:

Let's use a concrete example. Suppose you have a list...